workflows/ncn/hooks/before-all/install-csi.yaml

```diff
@@ -32,5 +32,23 @@ spec:
     source /srv/cray/scripts/metal/metal-lib.sh
     csi_url=$(paginate "https://packages.local/service/rest/v1/components?repository=csm-sle-15sp4" \
       | jq -r  '.items[] | .assets[] | .downloadUrl' | grep "cray-site-init" | sort -V | tail -1)
-    pdsh -S -w $(grep -oP 'ncn-\m\d+' /etc/hosts | sort -u | tr -t '\n' ',') "zypper install -y $csi_url"
+    if [[ -z $csi_url ]]; then
+      csi_url=$(paginate "https://packages.local/service/rest/v1/components?repository=csm-sle-15sp3" \
```

Comment:
Would it be easier for the user if instead of parsing these potential error messages if we just added CSI to both sp3 and sp4 repos in the tarball?